index.html

```diff
@@ -136,10 +138,12 @@ <h3>Interface Definition</h3>
     sequence&lt;ArrayBuffer&gt; getRemoteCertificates ();
     void                  start (RTCQuicParameters remoteParameters);
     void                  stop ();
-    RTCQuicStream         createStream (optional RTCQuicStreamParameters parameters);
+    RTCQuicBidirectionalStream         createBidirectionalStream ();
+    RTCQuicSendStream                  createSendStream ();
                     attribute EventHandler             onstatechange;
                     attribute EventHandler             onerror;
-                    attribute EventHandler             onquicstream;
+                    attribute EventHandler             onnewreceivestreamwithdata;
+                    attribute EventHandler             onnewbidirectionalstreamwithdata;

```

Comment:
Yeah, I was considering adding a comment about ideas for renaming this. We couldn't think of a better one, but I'm definitely open to other ideas.

Reply:
Presumably if you were implementing RTCDataChannel on top, you'd have some amount of header in the front of the stream (with something like the data channel ID).  That header would be the first data.

Reply:
The long name also seems awkward to me. I'm wondering why that information needs to be provided as part of the event's name. Is it important that the event only fires when there is data? In addition, is it important to emphasize that it's *new*?

How about `onreceivestream` and `onbidirectionalstream`?

Reply:
@lgrahl The event will only emit newly created streams, so “new” seems redundant. `onreceivestream` and `onbirectionalstream` seems more compact to me.

Also, the spirit of converging with WHATWG streams, should we support `cancel (reason)`?

Reply:
Yes, for converging with the Streams API, adding [`cancel`](https://streams.spec.whatwg.org/#rs-cancel) to the reader and [`abort`](https://streams.spec.whatwg.org/#ws-abort) to the writer would be sensible. IIRC, `.abortReading` and `.abortWriting` would just need to be renamed for that purpose. The Streams API also allows to return a `Promise`. This could for example be used to resolve once the closing procedure has been completed.

In my demo (which is based on bidirectional streams), `.abort` and `.cancel` trigger an error on both readable/writable controllers which results in exceptions on pending async read/write operations. For unidirectional streams, only the associated stream controller would have to be error-ed.

Reply:
I think we should leave them "abort" until we make a decision about WHATWG streams.  

This is similar to what I think of in general for this PR: let's merge it to get the main things in place and then address little things like the names with followup PRs.